examples/echo-api-key-simple.yaml

```diff
@@ -12,12 +12,18 @@ spec:
         key_selector: X-API-KEY # user defined
       api_key:
         label_selectors:
+          authorino.3scale.net/managed-by: authorino
           target: echo-api # user defined
   metadata: []
   authorization:
-    - JWTClaimSet:
-        match:
-          http:
-            path: "/api/*"
-        claim:
-          aud: "api"
+    - OPAPolicy: # only 5 min k8s secrets
+        uuid: e314f9a7-3381-4fe9-bc74-4bc02c7a666f
+        inlineRego: |
+          allow {
+            creationTimestampStr := object.get(object.get(identity, "metadata", {}), "creationTimestamp", {})
+            creationTimestamp := time.parse_rfc3339_ns(creationTimestampStr)
+            durationNs := time.now_ns() - creationTimestamp
+            durationSec := durationNs/1000000000
+            durationMin := durationSec/60
+            durationMin < 5
+          }
```

Comment:
As we discussed previously, maybe it would be better to extract this rego expiring key policy to a different file, and leave this one simpler with the non expiring key?